### PR TITLE
[Design2] DSCO - Colors - updated comments

### PR DESCRIPTION
**Pull request by levadadenys**
### [Design2] DSCO - Colors - updated comments

- updated comments for colors.css and primitiveColors.css DSCO Variables colors

While working on https://github.com/code-dot-org/code-dot-org/pull/62725 we've spotted that our DSCO colors documentation comments are outdated and it might be confusing to understand which colors should be used in which case. This PR updates these comments to give consumer the most fresh context on what these colors are and when they should be used.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by levadadenys** [comment]
Please let me know if there are any questions/ideas on how to improve this documentation here. I was working with these for a while, a fresh pair of eyes might be super helpful!

File: apps/src/componentLibrary/common/styles/colors.css [hunk 1]
```diff
@@ -1,17 +1,22 @@
 @import "primitiveColors.css";
 
-/* This file consists of Semantic and Utility colors,
- if you need DSCO colors, you need to import and use this file.
+/* This file consists of Semantic colors,
+ if you need DSCO colors with support of multiple themes, you need to import and use this file.
  */
 
-/* Colors are taken from DISCO Variables New Figma File - Semantic colors and Utility colors variables collections
+/* Colors are taken from DISCO Variables New Figma File - Semantic colors colors variables collection
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/common/styles/colors.css]: Should this be `DSCO`?
```suggestion
/* Colors are taken from DSCO Variables New Figma File - Semantic colors colors variables collection
```

File: apps/src/componentLibrary/common/styles/colors.css [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
  via variables2css plugin that generates css code based on Figma variables values and names.
  We're using css here in order to support not only scss, but also css files.
  Also, stylelint fix might be required for autogenerated colors, which can be done in one command.
  */
 
 /* Semantic Colors */
 
+/* Semantic colors values are taken from Primitive Colors (primitiveColors.css).
+ Semantic color value is specified for every theme, meaning that unlike (primitiveColors.css).
+ Semantic colors are meant to be different for every theme.
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/common/styles/colors.css]: Should this say `…unlike Utility colors (primitiveColors.css)`? Also made a tiny grammar fix.
```suggestion
 Semantic color value is specified for every theme, meaning that unlike Utility colors (primitiveColors.css), Semantic colors are meant to be different for every theme.
```

File: apps/src/componentLibrary/common/styles/primitiveColors.css [hunk 3]
```diff
@@ -1,5 +1,8 @@
-/* DO NOT IMPORT THIS FILE DIRECTLY. Use colors.css instead,
- which convey semantic and utility colors and also defines themes. */
+/* This is a collection of Primitive Colors.
+ Unlike semantic colors (color.css) this variables will be equal to same color value in all themes.
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/common/styles/primitiveColors.css]:
```suggestion
 Unlike Semantic colors (color.css) these variables will be equal to the same color value in all themes.
```

File: apps/src/componentLibrary/common/styles/primitiveColors.css [hunk 4]
```diff
@@ -1,5 +1,8 @@
...
+ Ideally, these colors should be used for components that are not theme dependent.
+ In the future it's possible that we will only need to use these colors in semantic colors (colors.css).
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/common/styles/primitiveColors.css]: Tiny consistency nit:
```suggestion
 In the future it's possible that we will only need to use these colors in Semantic colors (colors.css).
```

**Review by kelbyhawn — COMMENTED** [review]
Have a couple questions/nits, but thank you for updating this!

File: apps/src/componentLibrary/common/styles/colors.css (hunk 1 above)
Comment (kelbyhawn) [reply apps/src/componentLibrary/common/styles/colors.css]: Would it be worth linking to the Figma file here?

Comment (levadadenys) [reply apps/src/componentLibrary/common/styles/colors.css]: > Should this be DSCO?

Yep, thank you! Might had some mac t9 magic here

> Would it be worth linking to the Figma file here?

I asked design team if we want our Figma to be publicly available and answer was that we, at least for now, keep our working figma files private. So I think it's better not to include Figma file link for now

File: apps/src/componentLibrary/common/styles/colors.css (hunk 2 above)
Comment (levadadenys) [reply apps/src/componentLibrary/common/styles/colors.css]: > …unlike Utility colors (primitiveColors.css)

Utility colors are now deleted and deprecated. Primitive Colors are Primitive colors

Comment (kelbyhawn) [reply apps/src/componentLibrary/common/styles/colors.css]: Got it, thanks for clarifying!

Comment (kelbyhawn) [reply apps/src/componentLibrary/common/styles/colors.css]: Updated the original suggestion: 
```suggestion
 Semantic color value is specified for every theme, meaning that unlike Primitive Colors (primitiveColors.css), Semantic colors are meant to be different for every theme.
```

Comment (levadadenys) [reply apps/src/componentLibrary/common/styles/colors.css]: Thank you very much! 🤩

File: apps/src/componentLibrary/common/styles/colors.css (hunk 1 above)
Comment (kelbyhawn) [reply apps/src/componentLibrary/common/styles/colors.css]: Cool, that makes sense!

**Review by kelbyhawn — APPROVED** [review]
🎉

**Pull request merged**